frontend/src/views/ProjectsView.vue

```diff
@@ -1,16 +1,15 @@
 <template>
     <div class="projects">
+        <TitleContainer :title="$t('project.myProject')" class="titlecontainer">
+            <v-btn-toggle v-model="activeButton" class="button">
+                <v-btn value="finished">{{ $t("project.finished") }}</v-btn>
+            </v-btn-toggle>
+        </TitleContainer>
         <h1 v-if="isLoading">{{ $t("default.loading.loading_page") }}</h1>
-        <h1 v-else-if="isError || noProjectsFound" class="welcome">
+        <h1 v-else-if="isError || noProjectsFound" class="projectInfo">
```

Comment:
Wanneer er projecten geen projecten zijn, maar wel projecten die al verlopen zijn, is de 'no projects found' text niet zichtbaar, als dan de 'finished' knop geselecteerd wordt komen de oude projecten wel terug tevoorschijn